webapp/src/main/java/com/box/l10n/mojito/Application.java

```diff
@@ -115,4 +118,9 @@ public RetryTemplate retryTemplate() {
 
     return template;
   }
+
+  @Bean(name = "auditingDateTimeProvider")
+  public DateTimeProvider dateTimeProvider() {
+    return () -> Optional.of(OffsetDateTime.now());
```

Comment:
Why is this needed? Why is `OffsetDateTime` used here?

Reply:
I have changed to ZonedDateTime.

Reply:
Why is this bean needed? why is it fine to switch from `OffsetDateTime` to `ZonedDateTime `